src/data/markdown/translated-guides/en/04 Results output/200 Real-time/00 Prometheus remote write.md

```diff
@@ -158,14 +158,18 @@ k6 has special options for remote write output.
 | Name | Type |  Description |
 | ---- | ---- |  ----------- |
 | `K6_PROMETHEUS_RW_SERVER_URL` | `string`  | URL of the Prometheus remote write implementation's endpoint. Default is ``http://localhost:9090/api/v1/write`` |
-| `K6_PROMETHEUS_RW_HEADERS_<here-the-header-key>` | list of `string` | Additional headers to include in the HTTP requests. `K6_PROMETHEUS_RW_HEADERS_X-MY-HEADER=foo`|
-| `K6_PROMETHEUS_RW_USERNAME` | `string` | User for the HTTP Basic authentication at the Prometheus remote write endpoint. |
-| `K6_PROMETHEUS_RW_PASSWORD` | `string` | Password for the HTTP Basic authentication at the Prometheus remote write endpoint. |
-| `K6_PROMETHEUS_RW_PUSH_INTERVAL` | `string` | Interval of the metrics' aggregation and upload to the endpoint. Default is `5s` |
+| `K6_PROMETHEUS_RW_HEADERS_<here-the-header-key>` | list of `string` | Additional headers to include in the HTTP requests. `K6_PROMETHEUS_RW_HEADERS_X-MY-HEADER=foo`. |
```

Comment:
I spotted a couple of things here:
1. we refer to the parameter as a list of `string`, but we mention, and give an example, of using only a single one. Maybe we could rephrase to outline what I understand as being is meant: multiple headers of the same format with different `<header-key>` can be set in the environment.
2. In the example the `<header-key>` is prefixed with `_X` but not in the description, was it on purpose? 

```suggestion
| `K6_PROMETHEUS_RW_HEADERS_X-<header-key>` | `string` | Additional headers to include in the HTTP requests can be set using this described format. e.g. `K6_PROMETHEUS_RW_HEADERS_X-MY-HEADER=foo`. |
```

What do you think?

Reply:
```suggestion
| `K6_PROMETHEUS_RW_HEADERS_<header-key>` | `string` | Additional headers to include in the HTTP requests can be set using this described format. For example, `K6_PROMETHEUS_RW_HEADERS_MY-HEADER-CUSTOM-KEY=foo`. |
```